includes/admin/plugins.php

```diff
@@ -142,8 +142,8 @@ function perflab_render_plugin_card( array $plugin_data ) {
 	$version     = $plugin_data['version'];
 	$name        = wp_strip_all_tags( $title . ' ' . $version );
 
-	$requires_php = isset( $plugin_data['requires_php'] ) ? $plugin_data['requires_php'] : null;
-	$requires_wp  = isset( $plugin_data['requires'] ) ? $plugin_data['requires'] : null;
+	$requires_php = $plugin_data['requires_php'] ?? null;
+	$requires_wp  = $plugin_data['requires'] ?? null;
 
 	$compatible_php = is_php_version_compatible( $requires_php );
 	$compatible_wp  = is_wp_version_compatible( $requires_wp );
```

Comment:
Actually, I found out that the `requires_php` and `requires` are _always_ set. If they aren't present in the plugin, then they are returned in the API response as `false`. Take, for example, this ancient plugin which has no `Requires` nor `Requires PHP`: https://wordpress.org/plugins/automatic-glossary/

See the API response: https://api.wordpress.org/plugins/info/1.0/automatic-glossary.json

It includes:

```
  "requires": false,
  "requires_php": false,
  "requires_plugins": [],
```

Therefore, we can simplify this further. Instead of:

```php
	$requires_php = $plugin_data['requires_php'] ?? null;
	$requires_wp  = $plugin_data['requires'] ?? null;

	$compatible_php = is_php_version_compatible( $requires_php );
	$compatible_wp  = is_wp_version_compatible( $requires_wp );
```

This can rather be:

```php
	$compatible_php = ! $plugin_data['requires_php'] || is_php_version_compatible( $plugin_data['requires_php'] );
	$compatible_wp  = ! $plugin_data['requires'] || is_wp_version_compatible( $plugin_data['requires'] );
```

Technically it could be even simpler as:

```php
	$compatible_php = is_php_version_compatible( $plugin_data['requires_php'] );
	$compatible_wp  = is_wp_version_compatible( $plugin_data['requires'] );
```

However the phpdoc for these functions says it takes a `string` as input, while it actually works with `false` as well due to `empty()` checks.

Reply:
I've added an array shape to specify what the data looks like in another PR: #1175

Reply:
@westonruter update the changes as per request.